.changeset/early-ads-notice.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Fixes a bug in an unreleased PR, so it didn't feel necessary to have an entry for this. (#3786)